src/api/che.ts

```diff
@@ -534,5 +533,20 @@ export class CheHelper {
         entry.autodrain()
       }
     }
+
+    // Is a new project structure?
+    if (fs.existsSync(path.join(destDir, 'manager', 'manager.yaml'))) {
```

Comment:
Is it possible to make chectl know about two different structures?

Reply:
What do you mean?

Reply:
I meant not to move files, but use them directly. Just make chectl aware of the new files structure.
This is only an idea.